packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -38,8 +38,15 @@ import { invokeComponentCallback, invokeComponentRenderedCallback } from './invo
 import { Template } from './template';
 import { ComponentDef } from './def';
 import { LightningElement } from './base-lightning-element';
-import { startGlobalMeasure, endGlobalMeasure, GlobalMeasurementPhase } from './performance-timing';
-import { logOperationStart, logOperationEnd, OperationId, trackProfilerState } from './profiler';
+import { GlobalMeasurementPhase } from './performance-timing';
```

Comment:
Now that `performance-timing` is not used anywhere outside `profiler` shall we merge `performance-timing` and profiler code together?

Reply:
Done.